### PR TITLE
Better color sensor

### PR DESCRIPTION
**Pull request by hell0w0rld2**
Completed the tuning of the color sensor and adds an easier to read display.

### CODE REVIEW

File: components/colorSensor.py [hunk 1]
```diff
@@ -38,6 +38,16 @@ def getRed(self):
 
     def getBlue(self):
         return True if self.colorMatch.matchClosestColor(self.color, self.confidence) == self.colors["blue"] else False
+    
+    @feedback
+    def displayColor(self):
```
Comment (ChrisHirsch) [inline components/colorSensor.py:43]: Don't forget comments on a method

**Review by ChrisHirsch — DISMISSED** [review]
+1 unless you'd like to document the displayColor() method

**Review by ManiacMoustache — APPROVED** [review]
+1

**Review by Leaf-life — APPROVED** [review]
good

**Pull request merged**